src/mono/wasm/runtime/jiterpreter-jit-call.ts

```diff
@@ -281,15 +282,16 @@ export function mono_jiterp_do_jit_call_indirect(
                     jit_call_cb: jitCallCb,
                 },
                 m: {
-                    h: (<any>Module).asm.memory
+                    h: (<any>Module).getMemory()
```

Comment:
are the cryptic member names really worth it here ?

Reply:
Yes, they make the module header smaller